src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -59,6 +59,8 @@
     protected $startTime;
     protected $loadClassCache;
 
+    private $projectDir;
```

Comment:
Making things protected means we then have to ensure backward compatibility on the access to the property. This is a strong commitment. So we never replace private with protected without a strong use case for it. And if you have one, please open a dedicated issue to discuss it instead of doing it on the merged PR (people are generally looking at opened discussion only when catching up)